### ISSUE TITLE
Implement temporal masking

### ISSUE THREAD

**Issue by iver56** [asteroid-team/torch-audiomentations#44]
@nicofarr

**Comment by popcornell** [comment]
What do you mean by temporal masking ?
Can something like this be useful (basically dropout over only the last dimension ) ? 

```python 
class StepDrop(nn.Module):
  

    def __init__(self, p=0.5):
        self.p = p
        super().__init__()

    def forward(self, x):
        if not self.training or not self.p:
            return x

        binomial = torch.distributions.binomial.Binomial(probs=1 - self.p)
        mask = binomial.sample((x.size(0), x.size(-1))).to(x.device)
        mask = mask.div_(1 - self.p)
        mask = mask.unsqueeze(1)
        out = x * mask
        return out

```

**Comment by iver56** [comment]
I think @nicofarr means time mask like in SpecAugment, but applied to waveforms. I.e. pick one or more time spans and silence the audio there
